include/tesseract/baseapi.h

```diff
@@ -815,7 +815,7 @@ class TESS_API TessBaseAPI {
   EquationDetect* equ_detect_;     ///< The equation detector.
   FileReader reader_;              ///< Reads files from any filesystem.
   ImageThresholder* thresholder_;  ///< Image thresholding module.
-  GenericVector<ParagraphModel*>* paragraph_models_;
+  std::list<ParagraphModel*>* paragraph_models_;
```

Comment:
Also suspicious.
If vector worked here, why change to list?
Vector merges are find using .insert(end(), v2.begin(), v2.end());

Reply:
The code removes elements. That's expensive for `std::vector`, but cheap for `std::list`.

Reply:
I see. Legit then.
But I propose to optimize things at the end.
We try to update 1-to-1 code to STL containers first. Optimization will be at the very end.